### PR TITLE
fix: add padding to LoadingRow

### PR DESCRIPTION
**Pull request by vm**
This makes sure that there is no jump once the tokens are loaded

### CODE REVIEW

**Comment by vm** [comment]
This is the intended behavior, so this PR is reviewable.

File: src/components/Tokens/TokenTable/TokenRow.tsx [hunk 1]
```diff
@@ -346,7 +346,6 @@ export const HEADER_DESCRIPTIONS: Record<TokenSortMethod, ReactNode | undefined>
 /* Get singular header cell for header row */
 function HeaderCell({
   category,
-  sortable,
```
Comment (cmcewen) [inline src/components/Tokens/TokenTable/TokenRow.tsx:349]: why did you remove it from the destructuring and not the type?

File: src/components/Tokens/TokenTable/TokenRow.tsx [hunk 2]
```diff
@@ -441,7 +440,7 @@ export function HeaderRow() {
 }
 
 /* Loading State: row component with loading bubbles */
-export function LoadingRow() {
+export function LoadingRow(props: { first?: boolean; last?: boolean }) {
```
Comment (cmcewen) [inline src/components/Tokens/TokenTable/TokenRow.tsx:443]: can you use CSS rules for this?

Comment (vm) [reply src/components/Tokens/TokenTable/TokenRow.tsx:442]: i gave it a few tries and couldn't figure something out

**Review by cmcewen — APPROVED** [review]
